src/main/kotlin/calculator/NumberTokenizer.kt

```diff
@@ -0,0 +1,35 @@
+package calculator
+
+import calculator.vo.Delimiter
+import calculator.vo.Delimiters
+
+object NumberTokenizer {
+    private val DEFAULT_DELIMITERS = Delimiters.from(listOf(Delimiter(':'), Delimiter(',')))
+    private val CUSTOM_DELIMITER_SEARCH_PATTERN = Regex("""//(.)\\n(.*)""")
+
+    fun tokenize(input: String): List<Int> {
+        val (customDelimiter, separatedInput) = separateCustomDelimiterFrom(input)
+
+        val customizedDelimiters = DEFAULT_DELIMITERS.add(customDelimiter)
+
+        return customizedDelimiters
+            .split(separatedInput)
+            .map(this::parseToNumber)
+    }
+
+    private fun separateCustomDelimiterFrom(input: String): Pair<Delimiter?, String> {
+        val searchResult = CUSTOM_DELIMITER_SEARCH_PATTERN.find(input) ?: return null to input
```

Comment:
메서드를 연속으로 호출하는 경우 행구분을 해주세요~!

이 부분 뿐 아니라 다른 부분도 적용해주세요!

https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls

```suggestion
        val searchResult = CUSTOM_DELIMITER_SEARCH_PATTERN.find(input) 
            ?: return null to input
```